diff --git a/lib/analyzer.js b/lib/analyzer.js
index 96c52ee4845334e3a717758b9c18fafa0feb0cdf..025bbd375235e2f856c8bbf1ad7737a2bc2a99a3 100644
--- a/lib/analyzer.js
+++ b/lib/analyzer.js
@@ -23,7 +23,7 @@ const {
 } = require('./utils');
 
 const FILENAME_QUERY_REGEXP = /\?.*$/u;
-const FILENAME_EXTENSIONS = /\.(js|mjs|cjs)$/iu;
+const FILENAME_EXTENSIONS = /\.(js|mjs|cjs|bundle)$/iu;
 module.exports = {
   getViewerData,
   readStatsFromFile
